x/wasm/types/authz_test.go

```diff
@@ -744,3 +745,184 @@ func mustGrant(contract sdk.AccAddress, limit ContractAuthzLimitX, filter Contra
 	}
 	return *g
 }
+
+func TestValidateCodeGrant(t *testing.T) {
+	specs := map[string]struct {
+		codeHash              []byte
+		instantiatePermission AccessConfig
+		expErr                bool
+	}{
+		"all good": {
+			codeHash:              []byte("ABC"),
```

Comment:
nit: `"ABC"` is ok but if you want to be very verbose, let's make this something like `"any_valid_checksum"` (here and other placs)